setup.py

```diff
@@ -40,7 +40,7 @@
             fuse_examples_requirements.append(line.strip())
 
 # all extra requires
-all_requirements = fuseimg_requirements + fuse_requirements_dev
+all_requirements = fuseimg_requirements + fuse_requirements_dev + fuse_examples_requirements
```

Comment:
Not sure if it was on purpose without it.. It bothered me when I tried to run the EHR after installing a fresh env with `pip install -e .[all]`

Reply:
Yes, examples is a different package.
That's why it should be specified explicitly. pip install -e .[all, examples]

Reply:
reverting it